### PR TITLE
Added core_version_requirement lines to .info.yml files (d.o. issue 3151215)

### PR DESCRIPTION
**Pull request by joeparsons**
PR for [d.o. issue 3151215](https://www.drupal.org/project/cas_mock_server/issues/3151215).

### CODE REVIEW

File: cas_mock_server.info.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ name: CAS mock server
 type: module
 description: A mocked CAS server for testing purposes.
 core: 8.x
```
Comment (claudiu-cristea) [inline cas_mock_server.info.yml]: Drupal 8.7 support is over, also for security fixes. We should drop it too. Who wants to run the module will have to pin on 8.x-1.0-alpha3

File: tests/modules/cas_mock_server_test/cas_mock_server_test.info.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ type: module
 description: 'Test module for the CAS mock server.'
 package: Testing
 core: 8.x
```
Comment (claudiu-cristea) [inline tests/modules/cas_mock_server_test/cas_mock_server_test.info.yml]: Same here.

**Review by claudiu-cristea — CHANGES_REQUESTED** [review]

**Comment by pfrenssen** [comment]
I'm going to look into the test failure.

**Comment by pfrenssen** [comment]
Test was missing a dependency on the user module which is introduced in CAS 8.x-1.7. See #20

**Comment by pfrenssen** [comment]
Dropped the old style `core: 8.x` version requirement as suggested in https://www.drupal.org/docs/creating-custom-modules/let-drupal-know-about-your-module-with-an-infoyml-file

**Comment by pfrenssen** [comment]
@claudiu-cristea do you want to approve?

I am planning on also running tests on PHP 7.4 in a separate PR, then tomorrow we can roll a new release with D9 support.

**Review by claudiu-cristea — APPROVED** [review]

**Pull request merged**